src/components/App.js

```diff
@@ -487,7 +499,7 @@ class App extends React.Component {
               {
                 id: "network",
                 icon: networkIcon,
-                title: "Network",
+                title: "Network (only for Firefox)",
```

Comment:
Since we are going to use the client in demos, what about hiding the Network tab when we fail to subscribe to network events? "for Firefox only" could give the wrong impression, and this should automatically work on Chrome when they start supporting the events, which would make the label incorrect.

Reply:
True! Good idea! 
I'll fix it